src/EnergyPlus/SolarShading.cc

```diff
@@ -11946,7 +11946,7 @@ void CalcWinTransDifSolInitialDistribution(EnergyPlusData &state)
                                             construct_sh_BlAbsDiffBack(std::min(MaxSlatAngs, SurfWinSlatsAngIndex + 1), IGlass),
                                             SurfWinSlatsAngInterpFac);
                                     } else {
-                                        BlAbsDiffBk = construct_sh_BlAbsDiffBack(1, Lay);
+                                        BlAbsDiffBk = construct_sh_BlAbsDiffBack(1, IGlass);
```

Comment:
Original regression from https://github.com/NREL/EnergyPlus/commit/57aa34f2d6cd0cba59df94b5e49ddc98f9031bda#diff-18fbb8a5b1334b62513de2441bdb4bb4ef1fbe5fbec24d5dd0effcd019dfc33dR10986

I think It should be IGlass instead, fix looks good to me.